find-doi-requests/src/main/java/no/unit/nva/doi/requests/FindDoiRequestsHandler.java

```diff
@@ -52,14 +51,22 @@ public FindDoiRequestsHandler() {
     /**
      * Constructor for FindDoiRequestsHandler.
      *
-     * @param doiRequestsService    doiRequestsService
-     * @param environment   environment
+     * @param doiRequestsService doiRequestsService
+     * @param environment        environment
      */
     public FindDoiRequestsHandler(DoiRequestsService doiRequestsService, Environment environment) {
         super(Void.class, environment, logger);
         this.doiRequestsService = doiRequestsService;
     }
 
+    @Override
+    protected DoiRequestsResponse processInput(Void input, String apiGatewayInputString, Context context)
+        throws ApiGatewayException {
+        logger.info("Whole event below");
+        logger.info(apiGatewayInputString);
```

Comment:
if we want this type of logging, maybe just delegate the responsibility to the ApiGateway class that can be turned on and off by a configuration value?

Reply:
And maybe not info as a logger level. trace most likley.

Reply:
This actually should be deleted :)

Reply:
deleted.